src/senaite/patient/vocabularies.py

```diff
@@ -145,3 +145,25 @@ def __call__(self, context):
 
 
 EthnicitiesVocabularyFactory = EthnicitiesVocabulary()
+
+
+@implementer(IVocabularyFactory)
+class MaritalStatusesVocabulary(object):
+
+    def __call__(self, context):
+
+        marital_statuses = api.get_registry_record(
+            "senaite.patient.marital_statuses")
+
+        items = []
+        for marital_status in marital_statuses:
+            # note: the key will get the submitted value
+            keyword = marital_status.get("key")
+            title = marital_status.get("value")
+            # value, token, title
+            term = SimpleTerm(keyword, keyword, title)
+            items.append(term)
+        return SimpleVocabulary(sorted(items, key=lambda t: t.title))
```

Comment:
I think is better to remove this sorting, the "Up"/"Down" buttons from the records field (by means of the `allow_reorder` parameter) do not longer make sense otherwise:

![Captura de 2022-10-12 13-19-46](https://user-images.githubusercontent.com/832627/195329594-dddabb41-8a77-439d-a906-4bc9ed35ce8f.png)

Reply:
Ah, you are right!
We are sorting the vocabulary by title at the moment and overriding the manual sort order therefore.
Shall we just just drop the sorting in the vocabulary instead?

Reply:
> Shall we just just drop the sorting in the vocabulary instead?

Yes please

Reply:
Ok, I removed the alphabetical sorting now for all vocabularies and allow instead manual reordering of the items.